jupyter_notebooks/harmony_concise_api_test.ipynb

```diff
@@ -2,229 +2,453 @@
+    "    format=\"application/x-netcdf4\",\n",
+    ")\n",
+    "\n",
+    "request.is_valid()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b4e5544d",
+   "metadata": {},
+   "source": [
+    "Now that we have a valid request we simply need to call the submit function using the client we created earlier and pass in the request as a parameter.\n",
+    "\n",
+    "Tip: if you want to see the request before submitting it, use the request_as_curl function on the client to get an equivalent curl command for the request that will be submitted."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
+   "id": "36ce8016",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "curl -X GET -H 'Accept: */*' -H 'Accept-Encoding: gzip, deflate' -H 'Authorization: *****' -H 'Connection: keep-alive' -H 'Cookie: urs_user_already_logged=yes; token=*****; _urs-gui_session=441963edbfcbcaea1579d97ef9c7864a' -H 'User-Agent: Darwin/20.6.0 python-requests/2.26.0 harmony-py/0.4.2 CPython/3.8.10' 'https://harmony.earthdata.nasa.gov/C1940473819-POCLOUD/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&format=application%2Fx-netcdf4&maxResults=5&concatenate=true&skipPreview=true'\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(harmony_client.request_as_curl(request))\n",
+    "\n",
+    "job1_id = harmony_client.submit(request)"
+   ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "tags": []
-   },
+   "id": "9d095bf3",
+   "metadata": {},
    "source": [
-    "### Set up Environment and Collection data\n",
-    "\n",
-    "Below we set a default collection (C1940473819-POCLOUD) and environment (prod) which we will use for the podaac Concise test. As the field is tagged with parameters tag, when executing the notebook, they can be provided as inputs from command line, which will overwrite these default values.\n",
-    "Finding this information would complicate the tutorial- but po.daac has a tutorial available for using the CMR API to find collections and granules of interest. Please see the following tutorial for that information:\n",
+    "After submitting the request it is possible to retrieve the current processing status by using the job ID returned from the submission.\n",
     "\n",
-    "PODAAC_CMR.ipynb\n"
+    "If the request is still running, we can wait until the Harmony request has finished processing. This cell will wait until the request has finised."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "tags": [
-     "parameters"
-    ]
-   },
-   "outputs": [],
+   "execution_count": 6,
+   "id": "0d13a446",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "68421797-cd93-4cd6-b01a-4bf437c51f27\n",
```

Comment:
Could you update the text like this for better understandability?
"Example:
JobId: "68421797-cd93-4cd6-b01a-4bf437c51f27

Expected result:
{'status'...
"

Reply:
This is generated text from a print statement and not part of text.

Reply:
ok